pkgs/tools/security/signify/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, libbsd, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "signify-${version}";
+  version = "v24";
+
+  src = fetchurl {
+    url = "https://github.com/aperezdc/signify/archive/${version}.tar.gz";
+    sha256 = "0594vyvkq176xxzaz9xbq8qs0xdnr8s9gkd1prblwpdvnzmw0xvc";
+  };
+
+  doCheck = true;
+
+  buildInputs = [ libbsd pkgconfig ];
```

Comment:
```suggestion
  nativeBuildInputs = [ pkgconfig ];
  buildInputs = [ libbsd ];
```

Reply:
Cool - this forced me to learn the distinction here which has so far escaped me. It doesn't seem to be documented anywhere save a GitHub issues, which seems a bit of a shame.